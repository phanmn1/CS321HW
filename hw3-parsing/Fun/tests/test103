// the first character of a variable name is
// allowed to be uppercase
int count = 0;

int fact(int n) {
  Count = count + 1;
//^
  if (n<1)
    return 1;
  else
    return n * fact(n-1);
}

int x = y;
int y = 1;

void main() {
  print fact(7);
  print fact(3);
  print fact(count);
}

