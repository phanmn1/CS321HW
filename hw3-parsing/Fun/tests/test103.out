Lexical error at line 6, column 3.  Encountered: "C" (67), after : ""
